src/commands/generate/builder.js

```diff
@@ -33,7 +33,22 @@ const OPTIONS = {
     describe: 'Whether to run browser in headless mode',
     group: 'Launch'
   },
-
+  // Page configuration
+  'viewport-size': {
+    default: null,
+    type: 'string',
+    describe: 'Viewport size. Format: WIDTHxHEIGHT',
+    group: 'Page',
+    coerce: (viewportSize) => {
+      if (viewportSize) {
```

Comment:
I am not familiar with yargs, please feedback on this.

Reply:
This looks good to me! My only suggestion would be to break the `coerce` method into a helper file to make it easier to unit test.